doc/whats_new/v1.5.rst

```diff
@@ -18,6 +18,20 @@ For a short description of the main highlights of the release, please refer to
 Version 1.5.1
 =============
 
+**August 2024**
+
+Changelog
+---------
+
+:mod:`sklearn.linear_model`
+..........................
+
+-|Fix| Fixed sample weight handling for calculation of test scores under  _log_reg_scoring_path
+:pr:`29419` by :user:`Shruti Nath <snath-xoc>`
```

Comment:
Please only mention public API symbols in the changelog.

Also. Scikit-learn 1.5.1 is already released. The changelog entry should be moved to `v1.6.rst`.

Reply:
Here the most directly public symbol impacted by this fix is `LogisticRegressionCV`.